packages/wallets/src/components/WalletListCard/WalletListCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useWalletAccountsList } from '@deriv/api';
+import WalletBalance from '../WalletBalance/WalletBalance';
+import WalletCardIcon from '../WalletCardIcon/WalledCardIcon';
+import WalletListDetails from '../WalletListDetails/WalletListDetails';
+import IcDropdown from '../../public/images/ic-dropdown.svg';
+import './WalletListCard.scss';
+
+type TProps = {
+    account: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number];
+};
+
+const WalletListCard: React.FC<TProps> = ({ account }) => {
+    return (
+        <div className='wallet-list-header__card_container' key={account.loginid}>
```

Comment:
```suggestion
        <div className='wallets-list-header__card_container'>
```
@thisyahlen-deriv Class name in most files need to be updated from `wallet-` to `wallets-` 👀